src/stores/courseData.js

```diff
@@ -103,6 +104,14 @@ const store = new Vuex.Store({
       const classIndex = state.roads[state.activeRoad].contents.selectedSubjects[payload.classInfo.semester].indexOf(payload.classInfo);
       state.roads[state.activeRoad].contents.selectedSubjects[payload.classInfo.semester][classIndex].overrideWarnings = payload.override;
     },
+    setPASubstitutions (state, { uniqueKey, newReqs }) {
+      Vue.set(state.roads[state.activeRoad].contents.progressAssertions, uniqueKey, { 'substitutions': newReqs });
+      Vue.set(state.roads[state.activeRoad], 'changed', moment().format(DATE_FORMAT));
+    },
+    setPAIgnore (state, uniqueKey) {
```

Comment:
Would it be useful to keep the old substitutions around if you set it to ignore?  So that you could un-ignore it and get the old list of petitions back?  This would require adding a new button, so I'm not sure if it's worth it - would this be something likely to be used?

Reply:
If we made the Ignore button a toggle, then we wouldn't necessarily need an additional button. Now that you mention it, I could see it being inconvenient to lose your substitutions if you press the ignore button. I think it'd be a good idea to include.